### ISSUE TITLE
Dragonflight branch: Blizzard UI re-appearing upon reload

### ISSUE THREAD

**Issue by TruthNZ** [brittyazel/Neuron#446]
## Issue description:
Upon reloading, the Blizzard UI is re-appearing.
While re-enabling and disabling the option "Display the Blizzard UI" hides it again, it leaves up the window asking to ReloadUI (which of course makes the Blizzard UI re-appear.

Using the slash command /neuron BlizzUI does the same thing.

No lua errors occurring

## Technical info:
Version: Dragonflight branch from commit 2601463
Client: Retail 100000

**Comment by brittyazel** [comment]
yes, this branch is not complete nor even alpha. Neuron is on an indefinite hiatus as stated on Curseforge and Wowinterface.
